securedrop/journalist.py

```diff
@@ -188,6 +184,9 @@ def admin_add_user():
                 flash('There was an error with the autogenerated password. '
                       'User not created. Please try again.', 'error')
                 form_valid = False
+            except InvalidUsernameException as e:
+                form_valid = False
+                flash('Invalid username: ' + str(e), "error")
```

Comment:
In `admin_edit_user` we will also need an additional `elif` block to check if `new_username` is less than `Journalist.MIN_USERNAME_LEN`  - currently admins can set user passwords back to usernames that are too short.